Packages/Status/Sources/Status/Editor/StatusEditorViewModel.swift

```diff
@@ -172,6 +183,10 @@ public class StatusEditorViewModel: ObservableObject {
       visibility = status.visibility
       statusText = .init(string: mentionString)
       selectedRange = .init(location: mentionString.utf16.count, length: 0)
+
+      if !mentionString.isEmpty {
```

Comment:
Should this be reversed?

Reply:
Whoa I was not awake lol. Thanks

Reply:
This is checking that the local variable has data. When it does it sets the ivar mentionString.

Reply:
Yes, I was confused by the name shadowing at a quick glance.